apis_core/apis_entities/triple_configs/E21_PersonFromDNB.toml

```
@@ -0,0 +1,11 @@
+[[filters]]
+"rdf:type" = "gndo:DifferentiatedPerson"
+
+[attributes]
+forename = ["gndo:forename", "gndo:preferredNameEntityForThePerson/gndo:forename"]
+alternative_names = "gndo:variantNameForThePerson"
+surname = ["gndo:surname", "gndo:preferredNameEntityForThePerson/gndo:surname"]
+start_date_written = "gndo:dateOfBirth"
+end_date_written = "gndo:dateOfDeath"
```

```suggestion
date_of_birth = "gndo:dateOfBirth"
date_of_death = "gndo:dateOfDeath"
```
In the E21_Person class attributes were changed. Also in eg the wikidata toml also these attributes are used